index.html

```diff
@@ -83,136 +84,203 @@ <h2>Accessibility Map for COVID-19 assessment centres</h2>
                             </div>
                         </div>
                     </div>
-                    <div class="fl-mapviz-map"></div>
-                </div>
-                <div class="fl-mapviz-filter-panel">
-                    <div class="fl-mapviz-filters">
-                        <div class="fl-mapviz-filter fl-mapviz-filter-entrances">
-                            <span class="fl-mapviz-filter-icon"></span>
-                            <span class="fl-mapviz-filter-tooltip-icon fl-mapviz-hoverable"></span>
-                            <span class="fl-mapviz-filter-tooltip"></span>
-                            <label class="fl-mapviz-filter-title" for="toggle-1">Accessible Entrances</label>
-                            <label class="fl-checkbox-holder" tabindex="0">
-                                <input class="fl-mapviz-checkbox visually-hidden" tabindex="-1" id="toggle-1" type="checkbox" value="true" />
-                                <span></span>
-                            </label>
+
+                    <!-- A11y filter panel -->
+                    <div class="fl-mapviz-filter-panel">
+                        <div class="fl-mapviz-filters">
+                           <div class="fl-mapviz-filter fl-mapviz-filter-entrances">
+                               <span class="fl-mapviz-filter-icon"></span>
+                               <span class="fl-mapviz-filter-tooltip-icon flc-mapviz-hoverable"></span>
+                               <span class="fl-mapviz-filter-tooltip"></span>
+                               <label class="fl-mapviz-filter-title" for="toggle-1">Accessible Entrances</label>
+                               <label class="fl-checkbox-holder" tabindex="0">
+                                   <input class="fl-mapviz-checkbox visually-hidden" id="toggle-1" type="checkbox"/>
```

Comment:
The removal of tabindex=-1 from these elements has caused them to become spuriously keyboard focusable, but no focus indicator appears. The attribute should be put back